templates/kubernetes/managed.html

```diff
@@ -10,476 +13,442 @@
-        url="https://assets.ubuntu.com/v1/db240da7-K8s_management_platform-01.svg",
-        alt="Illustration K8s management platform",
-        width="260",
-        height="180",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-        }}
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>
+            Kubernetes monitoring
+            <br class="u-hide--medium u-hide--small" />
+            and analytics
+          </h2>
+        </div>
+        <div class="col">
+          <ul class="p-list--divided">
+            <li class="p-list__item is-ticked">
+              We build a robust cloud solution for you that provides dashboards for logging, monitoring and alerting.
+            </li>
+            <li class="p-list__item is-ticked">
+              We actively monitor your Kubernetes 24/7, providing observability, alerting, capacity planning and continuous service checks to ensure your cloud stays healthy.
+            </li>
+            <li class="p-list__item is-ticked u-no-padding--bottom">
+              We provide proactive management and visibility to help you gain a clear predictability over the costs, avoiding any surprises.
+            </li>
+          </ul>
+        </div>
       </div>
     </div>
+    <div class="u-fixed-width">
+      {{ image(url="https://assets.ubuntu.com/v1/b4915f70-monitoring and analytics.png",
+            alt="",
+            width="3696",
+            height="1541",
+            hi_def=True,
+            loading="lazy") | safe
+      }}
+    </div>
+  </section>
 
-    <hr class="p-rule is-fixed-width" />
-
-    <div class="row u-vertically-center">
-      <div class="col-7">
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
         <h2>Security and compliance</h2>
-        <ul>
-          <li>Every Kubernetes cluster built by Canonical is always upgradable to newer versions. Absolutely no snowflakes!</li>
-          <li>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">
+            Every Kubernetes cluster built by Canonical is always upgradable to newer versions. Absolutely no snowflakes!
+          </li>
+          <li class="p-list__item is-ticked">
             Zero downtime upgrades. Canonical guarantees upgrades to newer versions of Kubernetes on request. Nobody else does.
           </li>
-          <li>Regulatory compliance, ISO 27000 certified, GDPR compliant, SOC2 Type 2</li>
-          <li>
-            <a href="/blog/canonicals-managed-services-achieve-msp-cloud-verify-certification">MSP Cloud Verify certified</a> - the only managed service provider certification program accredited worldwide
+          <li class="p-list__item is-ticked">Regulatory compliance, ISO 27000 certified, GDPR compliant, SOC2 Type 2</li>
```

Comment:
Left a [q](https://chat.canonical.com/canonical/pl/rgreizrbgjbabk6odt5wrxmnew) about the punctuation on lists in general. Looks a bit weird that some of these have it and some don't